pkgs/applications/graphics/gscan2pdf/default.nix

```diff
@@ -17,6 +17,13 @@ perlPackages.buildPerlPackage rec {
     hash = "sha256-NGz6DUa7TdChpgwmD9pcGdvYr3R+Ft3jPPSJpybCW4Q=";
   };
 
+    patches = [
+    # fixes warnings during tests. See https://sourceforge.net/p/gscan2pdf/bugs/421
```

Comment:
This patch is rather large and only fixes warnings. Do we really need it?

Reply:
I switched to `fetchpatch`, so it shouldn't really matter for nixpkgs. 

Really necessary: No, but will probably be integrated in the next update and it makes output (and debugging) easier.